services/github/github-constellation.js

```diff
@@ -106,13 +103,15 @@ class GithubConstellation {
   }
 
   onTokenInvalidated(tokenString) {
-    process.nextTick(async () => {
-      try {
-        await this.persistence.noteTokenRemoved(tokenString)
-      } catch (e) {
-        log.error(e)
-      }
-    })
```

Comment:
This will also get in cases of local dev env/self-hosted that doesn't have redis configured but has set a GH token in the config which has hit the rate limits correct? I imagine this gets hit in non-persistence enabled scenarios which is why we wouldn't want to throw an error in this case (like above in `onTokenAdded`)?

Reply:
In practice we don’t generally hit the rate limit because of the `reserveFraction`, though IIRC `onTokenInvalidated` is not invoked for rate limit exceeded, only 403s. The reason to guard it here is that `this.persistence` might be unset, in which case this code would crash.